src/containers/BooksList.js

```diff
@@ -0,0 +1,23 @@
+import { useSelector } from 'react-redux';
+import Book from '../components/Book';
+
+const BookList = () => {
+  const books = useSelector(state => state.books);
```

Comment:
it uses `mapStateToProps` to fetch books from the `store` as recommended

Reply:
It should connect to the Redux store and fetch the books from the state (you can use the “connect()” function or Redux hooks).

I change it, but it specified that you can use either or.